src/hooks/transfers/useTransfers.tsx

```diff
@@ -87,14 +87,14 @@ export const useTransfers = () => {
   const debouncedAmountBN = useMemo(() => MustBigNumber(debouncedAmount), [debouncedAmount]);
 
   const chainsQuery = useQuery({
-    queryKey: ['transferEligibleChains', skipClientId],
+    queryKey: ['transferEligibleChains'],
```

Comment:
new instance of the skip client shouldnt trigger a re-query here since these are agnostic to whatever chain or logged in state the user is in

Reply:
wdym by logged in state? Previous iteration was purely if instance id changed?